Pod/Classes/MapperMacros/VOKManagedObjectMapperMacros.h

```diff
@@ -0,0 +1,74 @@
+//
+//  VOKManagedObjectMapperMacros.h
+//  Vokoder
+//
+//  Copyright © 2016 Vokal.
+//
+
+#ifndef VOKManagedObjectMapperMacros_h
+#define VOKManagedObjectMapperMacros_h
+
+#import "VOKManagedObjectMap.h"
+
+#import <VOKKeyPathHelper.h>
+
+/**
+ *  Creates a map with the default date mapper.
+ *
+ *  @param inputKeyPath           The foreign key to match with the local key.
+ *  @param coreDataSelectorSymbol The local selector symbol.
+ *  @param klass                  The class on which the local selector symbol is defined.
+ *
+ *  @return A VOKManagedObjectMap
+ */
+#ifndef VOKMapForeignToLocalClassProperty
+#   define VOKMapForeignToLocalClassProperty(inputKeyPath, klass, coreDataSelectorSymbol) \
+[VOKManagedObjectMap mapWithForeignKeyPath:inputKeyPath coreDataKey:VOKKeyForInstanceOf(klass, coreDataSelectorSymbol)]
```

Comment:
Were it me, I'd indent these macro line continuations to be one level deeper than the `define`, but Xcode will keep trying to put them back outdented to here, so it may not be worth it.